src/overlays/actors/ovl_En_Bigpo/z_en_bigpo.c

```diff
@@ -94,161 +153,1304 @@ static DamageTable D_80B65044 = {
+
+        Lights_PointNoGlowSetInfo(&firesPtr->info, thisx->home.pos.x, thisx->home.pos.y, thisx->home.pos.z, 0xFF, 0xFF,
+                                  0xFF, 0);
+    }
+
+    ActorShape_Init(&thisx->shape, 0.0f, func_800B3FC0, 45.0f);
+    thisx->bgCheckFlags |= 0x400;
+    this->savedHeight = thisx->home.pos.y + 100.0f;
+    this->mainColor.r = 0xFF;
+    this->mainColor.g = 0xFF;
+    this->mainColor.b = 0xD2;
+    this->mainColor.a = 0x00; // fully invisible
+
+    if ((this->switchFlags != 0xFF) && (Flags_GetSwitch(globalCtx, this->switchFlags))) {
+        Actor_MarkForDeath(&this->actor);
+    }
+
+    if (thisx->params == ENBIGPO_REGULAR) { // the well poe, starts immediately
+        thisx->flags &= ~0x10; // always update OFF
+        this->unkBool204 = true;
+        EnBigpo_InitWellBigpo(this);
+    } else if (thisx->params == ENBIGPO_SUMMONED) { // dampe type
+        EnBigpo_InitDampeMainPo(this);
+    }
+}
+
+void EnBigpo_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+    EnBigpo* this = (EnBigpo*)thisx;
+    GlobalContext* gCtx2;
+    s32 fireCount;
+
+    if ((this->actor.params != ENBIGPO_POSSIBLEFIRE) && (this->actor.params != ENBIGPO_CHOSENFIRE) &&
+        (this->actor.params != ENBIGPO_REVEALEDFIRE) && (this->actor.params != ENBIGPO_UNK5)) {
+        // if NOT a fire type, ENBIGPO_REGULAR and ENBIGPO_SUMMONED combat types only
+        if (1) {}
+        gCtx2 = globalCtx;
+        for (fireCount = 0; fireCount < 3; ++fireCount) {
+            LightContext_RemoveLight(gCtx2, &gCtx2->lightCtx, this->fires[fireCount].light);
+        }
+        Collider_DestroyCylinder(gCtx2, &this->collider);
+    }
+}
+
+void func_80B61914(EnBigpo* this) {
+    EnBigpoFireParticle* firePtr;
+    s32 i;
+
+    for (i = 0; i < 3; i++) {
+        firePtr = &this->fires[i];
+        firePtr->pos.x = (Math_SinS(this->actor.shape.rot.y) * this->fireRadius) + this->actor.world.pos.x;
+        firePtr->pos.z = (Math_CosS(this->actor.shape.rot.y) * this->fireRadius) + this->actor.world.pos.z;
+        this->actor.shape.rot.y += 0x5555;
+    }
+}
+
+void EnBigpo_UpdateSpin(EnBigpo* this) {
+    s16 oldYaw = this->actor.shape.rot.y;
+    this->actor.shape.rot.y += this->rotVelocity;
+    if ((oldYaw < 0) && (this->actor.shape.rot.y > 0)) {
+        Audio_PlayActorSound2(&this->actor, NA_SE_EN_PO_ROLL); // spinning sfx during spin attack
+    }
+}
+
+// force camera to.. what?
+void func_80B619FC(EnBigpo* this, GlobalContext* globalContext) {
+    Camera* cam;
+    if (this->cutsceneCam != NULL) {
+        cam = Play_GetCamera(globalContext, this->cutsceneCam);
+        cam->eye.y -= this->actor.velocity.y;
+        if (this->actor.velocity.y > 0.0f) {
+            cam->eye.x -= 1.5f * Math_SinS(this->actor.yawTowardsPlayer);
+            cam->eye.z -= 1.5f * Math_CosS(this->actor.yawTowardsPlayer);
+        }
+        func_8016970C(globalContext, this->cutsceneCam, &this->actor.focus.pos, &cam->eye);
+    }
+}
```

Comment:
```suggestion
// Lowers the position/eye of the camera during the Big Poe spawn cutscene
void EnBigpo_LowerCutsceneSubCamera(EnBigpo* this, GlobalContext* globalContext) {
    Camera* subCamId;
    if (this->cutsceneSubCamId != MAIN_CAM) {
        subCamId = Play_GetCamera(globalContext, this->subCamId);
        subCamId->eye.y -= this->actor.velocity.y;
        if (this->actor.velocity.y > 0.0f) {
            subCamId->eye.x -= 1.5f * Math_SinS(this->actor.yawTowardsPlayer);
            subCamId->eye.z -= 1.5f * Math_CosS(this->actor.yawTowardsPlayer);
        }
        func_8016970C(globalContext, this->subCamId, &this->actor.focus.pos, &subCamId->eye);
    }
}
```
So what this function is doing is lowering the camera. Eye is the position of the camera. 

https://youtu.be/_cJIc_KWrxs?t=572

Do you notice that as soon as the flames touch, the camera starts to lower? That's what this function is doing, it's updating the camera coordinates. Perhaps you can thing of a better name then me, but that's the idea